reconstruction/ec/src/main/java/org/jlab/service/ec/ECCommon.java

```diff
@@ -150,11 +151,12 @@ public static List<ECStrip>  initEC(DataEvent event,  ConstantsManager manager){
         
         isMC = run<=100;
         
-        if(isMC) {usePass2Timing = false; useDTCorrections = false; useFTpcal = false;}
+        if(isMC) {usePass2Timing = false; usePass2Energy = false; useDTCorrections = false; useFTpcal = false;}
```

Comment:
To move away from this iff statement, can the new attenuation table be populated for runs <100 so that the new formula can be used for MC?

Reply:
Done.